### PR TITLE
Move VST plugin code into main repo

### PR DESCRIPTION
**Pull request by mikeoliphant**

### CODE REVIEW

File: LiveSPICEVst/EditorView.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+using Microsoft.Win32;
+using SharpSoundDevice;
+using System;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace LiveSPICEVst
+{
+    /// <summary>
+    /// Dummy VST host class to use the the UI Test App
```
Comment (dsharlet) [inline LiveSPICEVst/EditorView.xaml.cs:11]: Can this class also go in the MockVst app? Or is it needed as a placeholder or something? If that's the case, can you update the comment to clarify?

**Review by dsharlet — COMMENTED** [review]
Thank you for sharing this! This mostly looks great, I have just one question about the DummyHostInfo, trying to better understand the structure of the code. 

Tagging #16

**Comment by mikeoliphant** [comment]
Yeah, it shouldn't be too hard to move it - it will just take some
refactoring of how the MockVst creates the EditorView. If I update my
branch, will it automatically update the pull request?

**Comment by dsharlet** [comment]
I believe so, yes. I already see a new commit re: rebuilding the circuit when buttons change.

**Comment by mikeoliphant** [comment]
Yeah - I went ahead and committed on my branch. I fixed the issue I was
having with bypass buttons, and moved the DummyHostInfo class into the
MockVst project.

**Comment by dsharlet** [comment]
Oh nice, I see that now. Thanks for that change, I know it seems minor but it helps me understand the structure of the plugin and the test better.

I think this is ready to merge :) I did see some other things (e.g. error handling in the case a circuit fails to load, etc.) but I figured it's easier to address these things as separate changes, this is still a separate branch for now after all.

**Comment by mikeoliphant** [comment]
Yep - it is by no means "finished". A pass at error handling would
definitely be useful.

**Pull request merged**